docs/source/deployguide/ca-config.md

```diff
@@ -0,0 +1,417 @@
+
+## operations
+
+```
+operations:
+    # host and port for the operations server
+    listenAddress: 127.0.0.1:9443
+
+    # TLS configuration for the operations endpoint
+    tls:
+        # TLS enabled
+        enabled: false
+
+        # path to PEM encoded server certificate for the operations server
+        cert:
+            file:
+
+        # path to PEM encoded server key for the operations server
+        key:
+            file:
+
+        # require client certificate authentication to access all resources
+        clientAuthRequired: false
+
+        # paths to PEM encoded ca certificates to trust for client authentication
+        clientRootCAs:
+            files: []
+```
+
+The operations service can be used for health monitoring of the CA and relies on mutual TLS for communication with the node.
+Therefore, you need to set `operations.tls.clientAuthRequired` to `true`. When this is set to `true`, clients attempting to ascertain the health of the node are  required to provide a valid certificate for authentication. If the client does not provide a certificate or the service cannot verify the client’s certificate, the request is rejected. This means that the clients will need to register with the TLS CA and provide their TLS signing certificate on the requests.
+
+In the case where two CAs are running on the same machine, you need to modify the `listenAddress:` for the second CA to use a different port. Otherwise, when you start the second CA, it will fail to start, reporting that `the bind address is already in use`.
+
+## metrics
+
+```
+metrics:
+    # statsd, prometheus, or disabled
+    provider: disabled
+
+    # statsd configuration
+    statsd:
+        # network type: tcp or udp
+        network: udp
+
+        # statsd server address
+        address: 127.0.0.1:8125
+
+        # the interval at which locally cached counters and gauges are pushsed
+        # to statsd; timings are pushed immediately
+        writeInterval: 10s
+
+        # prefix is prepended to all emitted statsd merics
+        prefix: server
+```
+
+If you want to monitor the metrics for the CA, choose your metrics provider:
+
+- **provider**: `Statsd` is a push model, `Prometheus` is a pull model. Because Prometheus is a pull model there is not any configuration required from the Fabric CA server side. Rather, Prometheus sends requests to the operations URL to poll for metrics. [Available metrics](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/metrics_reference.html).
+
+## Next steps
+
+After deciding on your CA configuration, you are ready to deploy your CAs. Follow instructions in the Create a CA tutorial to start your CA.
```

Comment:
Are you referring to the CA deployment steps? I would not use the word tutorial since it is not in the tutorial section.

Reply:
Done